tests/test_barplot.py

```diff
@@ -50,55 +41,48 @@ def test_teba002_change_axis_value(dash_duo):
 
     x.send_keys("model-year")
     x.send_keys(Keys.RETURN)
-
-    time.sleep(1)
+    time.sleep(0.5)
 
     assert "model-year" in driver.find_element(By.CLASS_NAME, "xtitle").text
     assert dash_duo.get_logs() == [], "browser console should contain no error"
 
     driver.close()
 
 
-def test_teba003_set_cluster(dash_duo):
-    driver = dash_duo.driver
-    dash_duo.start_server(setup_xiplot_dash_app(data_dir="data"))
-    time.sleep(1)
-    dash_duo.wait_for_page()
+# def test_teba003_set_cluster(dash_duo):
```

Comment:
Why is this test commented out?

Reply:
The cluster column doesn't exist by default anymore, so this test would need a major rework (there is a comment about it a couple rows down). I thought I would be a bit lazy, but if necessary I can take a look at it.

Reply:
The test is now working